src/pages/CloudDeployProgress.js

```diff
@@ -15,8 +15,14 @@
 import React from 'react';
 
 import { translate } from '../localization/localize.js';
-import { PRE_DEPLOYMENT_PLAYBOOK, DAYZERO_SITE_PLAYBOOK,
-  SITE_PLAYBOOK, STATUS } from '../utils/constants.js';
+import {
+  PRE_DEPLOYMENT_PLAYBOOK, DAYZERO_SITE_PLAYBOOK,
+  CONFIG_PROCESSOR_RUN_PLAYBOOK, READY_DEPLOYMENT_PLAYBOOK,
+  NETWORK_INTERFACE_DEPLOY_PLAYBOOK, NOVA_DEPLOY_PLAYBOOK,
+  IRONIC_DEPLOY_PLAYBOOK, MAGNUM_DEPLOY_PLAYBOOK, ARDANA_STATUS_PLAYBOOK,
+  MONASCA_AGENT_DEPLOY_PLAYBOOK, MONASCA_DEPLOY_PLAYBOOK,
+  SWIFT_DEPLOY_PLAYBOOK, MONASCA_TRANSFORM_DEPLOY_PLAYBOOK, CEPH_DEPLOY_PLAYBOOK,
+  CINDER_DEPLOY_PLAYBOOK, SITE_PLAYBOOK, STATUS } from '../utils/constants.js';
```

Comment:
might be worth while doing a `import * as constants from '../utils/constants';` That way reference can be something like `constants.CONFIG_PROCESSOR_RUN_PLAYBOOK` and we don't have to keep this huge list here.

Reply:
@eapodaca  good idea ...will do